### ISSUE TITLE
Kill running timer upon receiving Alexa off command

### ISSUE THREAD

**Issue by LankyGit1** [Aircoookie/WLED#2421]
If you have an API command defined that has a timer as part of its functionality eg sunrise or sunset then if wled receives an off command from Alexa there should have the ability to abort the timer. As currently the leds will switch off then continue with the running timer command for example increasing intensity.
Whether this functionality should have a switchable option of working as it currently does ie the timer will keep running or abort I don't mind.

Unless I am missing another way of doing this?

Thanks

**Comment by blazoncek** [comment]
Can you be more specific what kind of timer are you talking about?

**Comment by LankyGit1** [comment]
I have the following API command preset triggered at a specific time each day. If I decide to abort the preset by using Alexa to turn off, the leds turn off but then as the timer is still running on the wled interface the leds start to illuminate again. Sorry can't remember where I got this API command from originally but I think I have tweaked the colours also.

&FX=0&CL=hff0000&C2=hffb845&A=5&NL=20&NT=255&NF=2
